packages/puppeteer-core/src/node/ChromeLauncher.ts

```diff
@@ -266,3 +270,23 @@ function convertPuppeteerChannelToBrowsersChannel(
       return BrowsersChromeReleaseChannel.CANARY;
   }
 }
+
+/**
+ * Extracts all features from the given command-line flag.
+ *
+ * Example input:
+ * ["--enable-features=NetworkService,NetworkServiceInProcess", "--enable-features=Foo"]
+ *
+ * Example output:
+ * ["NetworkService", "NetworkServiceInProcess", "Foo"]
+ */
+export function getFeatures(flag: string, options: string[] = []): string[] {
+  const opts = options
+    ?.filter(s => {
```

Comment:
Options is always an array. So we can remove the `?` and return the `options.filter().map()` directly